tests/test-rate-limits.sh

```
@@ -32,8 +35,15 @@ test_worker() {
     for i in $(seq 1 $TEST_COUNT); do
         echo -n "Request $i/$TEST_COUNT: "
         
-        # Make the request and capture both response body and status code
-        response=$(curl -s -w "%{http_code}" -o /tmp/response_$i.json "$endpoint" 2>/dev/null)
+        # Make the request with unique agent headers and capture body + status code
+        response=$(curl -s \
+          -H "x-agent: $AGENT_ID" \
+          -H "x-Agent: $AGENT_ID" \
+          -H "User-Agent: prisma-rate-limit-test/$AGENT_ID" \
+          -w "%{http_code}" \
+          -o /tmp/response_$i.json \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)